### PR TITLE
nsqd: bump go-options / log when we can't get channels for pre-creation

### PR DESCRIPTION
**Pull request by mreiferson**
Addresses proposal (1) and (2) of #826

### CODE REVIEW

**Comment by mreiferson** [comment]
RFR @jehiah 

cc @ploxiln @stephensearls

File: nsqd/nsqd.go [hunk 1]
```diff
@@ -444,7 +444,10 @@ func (n *NSQD) GetTopic(topicName string) *Topic {
 	// this makes sure that any message received is buffered to the right channels
 	lookupdHTTPAddrs := n.lookupdHTTPAddrs()
 	if len(lookupdHTTPAddrs) > 0 {
-		channelNames, _ := n.ci.GetLookupdTopicChannels(t.name, lookupdHTTPAddrs)
+		channelNames, err := n.ci.GetLookupdTopicChannels(t.name, lookupdHTTPAddrs)
+		if err != nil {
+			n.logf("ERROR: failed to get channels for pre-creation for topic %s - %s", t.name, err)
```
Comment (jehiah) [inline nsqd/nsqd.go]: Technically we failed to query all lookupd's so we don't know if the result is deficient in any way. We may have still gotten channels for pre-creation from available lookupd's. Rephrase this and/or make different error messages based on `channelNames`?

Comment (mreiferson) [inline nsqd/nsqd.go]: Fair point, will update 👍

Comment (mreiferson) [inline nsqd/nsqd.go]: how's this?

**Pull request merged**